woff2/index.html

```diff
@@ -886,6 +899,51 @@ <h3 id="woff20Header-0"><span class='secno'>3.2. </span>WOFF2 Header</h3>
       where data from various separate substreams are recombined to create a complete glyph
       record for each entry of the original <span class=tt>glyf</span> table.</p>
 
+      <div class="correction" id="c2">
+        <span class="marker">Candidate Correction 2:</span>
+          <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/glyf">Simple glyph flags</a> define the format of x/y coordinate vectors for each
+          point of the glyph outline. In WOFF2, the pre-processing step converts both
+          the flags and the coordinates themselves into different data streams. Six
+          different flags (bits 0-5) can be set to define on/off-curve point and the
+          format for each set of point coordinates, flags can be shared among multiple
+          point coordinates.
+        <p>The bit 6 flag (OVERLAP_SIMPLE) is different, it can only be set once on the
+          first flag of the glyph. If set, it indicates that contours of the glyph
+          _may_ overlap. The OpenType spec doesn't require this flag be set, and some
+          (but not all) rasterizers would ignore this bit. Bit7 is reserved / unused.</p>
+        <p>Since
+          flag bit6 is not required and bit 7 is reserved,
+          in the original WOFF2 Recommendation they could simply be omitted
+          when the source data wass encoded, and set to '0' upon reconstruction of glyph
+          data. Apparently, this has caused the loss of data for flag bit 6 in some
+          fonts that have glyphs with overlapping contours where this optional bit 6 is set,
+          and it's an issue for some rasterizers that still rely on flag bit 6 setting
+          to deal with contour overlaps.</p>
+        <p>This Candidate Correction solves the problem, firstly,
+          by redefining <a href="#newflags">a previously 16-bit reserved field</a>
+          in the <a href="#woff20Header">WOFF2Header</a>.
```

Comment:
We did initially discuss reusing the reserved field in the WOFF2Header as an option to fix this issue, but this is not the solution we agreed to in the end. The WG decision was that it would be better to have the fix localized to the "Transformed 'glyf' table", and have this new options field defined as part of the transformed 'glyf' table itself. 
The [exact changes](https://lists.w3.org/Archives/Public/public-webfonts-wg/2021May/0011.html) discussed and approved by the group suggest replacing the `Fixed version` field with `UInt16 version`, (because the `version` field in its current form has never been defined and/or used) and reusing the remaining (freed) Uint16 field as optionFlags, where one flag is defined to indicate the presence of the new overlapSimpleBitmap[] and all other flags are reserved.

Reply:
Totally missed that, was editing based on the earlier proposal. Will fix, though likely not before today's call.

Reply:
Correct changes edited in.
Also marked up two AT conformance clauses and three UA conformance clauses.